drivers/spi/spi_context.h

```diff
@@ -287,6 +289,21 @@ static inline void *spi_context_get_next_buf(const struct spi_buf **current,
 	return NULL;
 }
 
+static inline size_t spi_stm32_count_total_bytes(const struct spi_buf_set *bufs)
```

Comment:
```suggestion
static inline size_t spi_count_total_bytes(const struct spi_buf_set *bufs)
```
And this modification should be done in a dedicated commit with proper justification

Reply:
This modification is included in the "Use SPI FIFO" commit because it's related to it: it's necessary to decide if the SPI FIFO can be used. The other commits could be considered as related, but they are useful by themselves, unlike this one, that would be only adding an extra parameter which, if not used, only adds complexity to the code.

Reply:
First issue here is that you're modifying a generic driver file using vendor prefix. This isn't allowed.
Then it is good practice in general to split API changes from vendor specific implementations (it could be useful for instance a revert has to be done on the vendor code modification).

Reply:
Done.